exporter/importables.go

```diff
@@ -2294,4 +2294,40 @@ var resourcesMap map[string]importable = map[string]importable{
 		},
 		// TODO: add Depends & Import to emit corresponding UC Volumes when support for them is added
 	},
+	"databricks_storage_credential": {
+		WorkspaceLevel: true,
```

Comment:
We also may need to define `ShouldOmitField` otherwise some fields marked as `computed` (i.e., `owner`, ...) will be skipped when generating the code.